include/quicly/cc.h

```diff
@@ -0,0 +1,38 @@
+/* Interface definition for quicly's congestion controller.
+ */
+
+#ifndef quicly_cc_h
+#define quicly_cc_h
+
+#include "quicly/constants.h"
+#include <assert.h>
+#include <stdint.h>
+#include <string.h>
+
+typedef struct st_quicly_cc_t {
+    uint32_t cwnd;
+    uint32_t ssthresh;
+    uint32_t stash;
+    uint64_t recovery_end;
+} quicly_cc_t;
+
+void quicly_cc_init(quicly_cc_t *cc);
+
+/* Called to query the controller whether data can be sent. Returns 1 if yes, 0 otherwise.
```

Comment:
Would you mind changing this to:
```c
/**
 * comment
 */
```

We generally try to use the doxygen style doc-comments (though I've never applied doxygen!).